src/components/Kv/KvLoadingPlaceholder.vue

```diff
@@ -0,0 +1,37 @@
+<template functional>
```

Comment:
These KvLoading components are sweet. When time can you add some .stories please.

Reply:
👍